pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    # truncated_range,
+    # truncated_discrete_set,
+    strict_discrete_set,
+)
+# from pymeasure.adapters import VISAAdapter
+from .buffer import KeithleyBuffer
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the socket is hardcoded to 2268,
+        example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+        the read termination is \n
+
+    .. code-block:: python
+
+        source = Keithley2260B("GPIB::1")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+
+    """
+
+    enabled = Instrument.control(
+        "OUTPut?", "OUTPut %d",
+        """A boolean property that controls whether the source is enabled, takes
+        values True or False (1 or 0). """,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True
```

Comment:
If you put a trailing `,` here, if you add another argument in the future you will not have to touch this line, too (to add it). This makes the git log/blame clearer.

Reply:
thanks for the suggestion!